### ISSUE TITLE
Don't fail-fast and increase concurrent jobs for matrix jobs

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-build-scripts#25]
https://help.github.com/en/articles/workflow-syntax-for-github-actions

`jobs.<job_id>.strategy.fail-fast`

When set to true, GitHub cancels all in-progress jobs if any matrix job fails. Default: true

`jobs.<job_id>.strategy.max-parallel`

The maximum number of jobs that can run simultaneously when using a matrix job strategy. By default, GitHub will maximize the number of jobs run in parallel depending on the available runners on GitHub-hosted virtual machines.

```
strategy:
  max-parallel: 2
```

**Comment by CAMOBAP795** [comment]
Already done 83e54455d5fa57c3fa33abbf0e296b52957eb24f
